### PR TITLE
minor: add sponsoring badges

### PR DESCRIPTION
**Pull request by romani**
continuation of #5946

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,20 @@ Questions and Answers from community: [![][stackoverflow img]][stackoverflow]
 
 Bugs and Feature requests (not the questions): https://github.com/checkstyle/checkstyle/issues
 
+
+Support/Sponsor checkstyle
+========
+
+If you want to speed up fixing of issue and want to encourage somebody in
```
Comment (rnveach) [inline README.md]: `in` => `on the`

File: README.md [hunk 2]
```diff
@@ -124,5 +138,20 @@ directory.
...
+
+[salt.bountysource]:https://salt.bountysource.com/teams/checkstyle
+[salt.bountysource img]:https://img.shields.io/bountysource/team/checkstyle/activity.svg?label=salt.bountysource
+
+[backers.opencollective]:https://opencollective.com/checkstyle/
+[backers.opencollective img]:https://opencollective.com/checkstyle/backers/badge.svg
 
-              
+[sponsors.opencollective]:https://opencollective.com/checkstyle/
+[sponsors.opencollective img]:https://opencollective.com/checkstyle/sponsors/badge.svg
```
Comment (rnveach) [inline README.md]: I can't tell for sure from web GUI. Does this file still end with an empty line?

File: README.md (hunk 1 above)
Comment (romani) [reply README.md]: fixed.

File: README.md (hunk 2 above)
Comment (romani) [reply README.md]: confirmed, there is empty line at the end.

**Comment by romani** [comment]
merged as https://github.com/checkstyle/checkstyle/commit/83ba4bac157d13a63ced0217e5f478bb539afefe

**Pull request closed without merging**